### DISCUSSION TITLE
Is there a plan or schedule for Flutter package to support Windows platform?

### DISCUSSION THREAD

**Discussion by FirokOtaku** [Tencent/MMKV#1435] [General]
Currently [MMKV for Flutter](https://pub.dev/packages/mmkv) is only supporting Android and iOS. Will the Windows platform be officially supported in the future?

**Comment by lingol** [comment]
Not planning on this. Why would you use flutter on windows or any non mobile platforms anyway?

**Reply by FirokOtaku** [reply]
Umm, Flutter as a modern cross-platform framework has official Windows support, that's it.
And even mainly targeted platform is Android/iOS, sometimes we just want to run the program on local desktop machine for testing or something.

Anyway, thanks for that information. I will consider other ways to handle data on Windows platform.

**Comment by lingol** [comment]
It's supported in [v2.2.3](https://github.com/Tencent/MMKV/releases/tag/v2.2.3).
